why-django.qmd

```diff
@@ -157,8 +138,18 @@ What decision was made, use the form "We decided on CHOICE because of
 REASONS."
 :::
 
-After evaluating Flask, FastAPI, Ruby on Rails, and Django, we have made
-the decision that Django is the better fit for Seedcase. Django has the
-benefits of a built-in serialization framework and REST framework for
-API development, providing a powerful ORM and good built-in support for
-security, as well as user management.
+We decided to choose Django because the requirements of Seedcase are a close match for what Django offers, so we are well-placed to take advantage of many of its core features. Django's built-in ORM, extensive user management capabilities, feature-rich REST framework and solid security principles will save us significant development time and effort and help us create a reliable and long-lived product. The native template engine is also attractive, in that it will allow us to iterate over user interfaces quickly, without the overhead of maintaining a separate frontend framework.
```

Comment:
```suggestion
We decided on Django because the requirements of Seedcase are a close match for what Django offers, so we are well-placed to take advantage of many of its core features. Django's built-in ORM, extensive user management capabilities, feature-rich REST framework, and solid security principles will save us significant development time and effort and help us create a reliable and long-lived product. The native template engine is also attractive, in that it will allow us to iterate over user interfaces quickly, without the overhead of maintaining a separate front-end framework.
```